Tools/MonoGame.Content.Builder/MonoGame.Content.Builder.csproj

```
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
```

We can disable the warning for having no assembly for the TFM, see https://github.com/nuget/home/issues/8583.
```suggestion
    <IncludeBuildOutput>false</IncludeBuildOutput>
    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
```

---

Added.